subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/java/JavaIncrementalExecutionPerformanceTest.groovy

```diff
@@ -49,7 +49,7 @@ class JavaIncrementalExecutionPerformanceTest extends AbstractIncrementalExecuti
     }
 
     @RunFor([
-        @Scenario(type = PER_COMMIT, operatingSystems = LINUX, testProjects = ["largeGroovyMultiProject", "largeMonolithicGroovyProject"], iterationMatcher = "assemble for non-abi change"),
+        @Scenario(type = PER_COMMIT, operatingSystems = LINUX, testProjects = ["largeGroovyMultiProject", "largeMonolithicJavaProject", "largeMonolithicGroovyProject"], iterationMatcher = "assemble for non-abi change"),
```

Comment:
Is there a regression in `largeMonolithicJavaProject` as well? Can you add this to the PR description?

Reply:
Done. Also, for full tracking pls. see https://github.com/gradle/gradle-private/issues/3623.